src/app/sliders/horizontal-slider/horizontal-slider.component.ts

```diff
@@ -138,6 +153,21 @@ export class HorizontalSliderComponent
         this.sliderResizeObserver.disconnect();
     }
 
+    resizeSliderAndThumbRaduis() {
+        this.screenHeight = window.innerHeight;
+        this.screenWidth = window.innerWidth;
+        this.trackHeight =
+            this.trackHeightSave *
+            ((this.screenHeight + this.screenWidth) / 10000 + 1);
+        this.thumbRadius =
+            this.thumbRadiusSave *
+            ((this.screenHeight + this.screenWidth) / 10000 + 1);
+        if (this.screenHeight + this.screenWidth >= 2900) {
+            this.trackHeight = this.trackHeight * 1.3;
+            this.thumbRadius = this.thumbRadius * 1.3;
+        }
+    }
+
```

Comment:
Can this be done via CSS functions/is there a specific reason for the numbers or where they choosen because they happen to look good?